code-challenges.test.js

```diff
@@ -11,6 +11,16 @@
 // Run the file with the following command: $ yarn jest
 
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
+// Psuedo code:
+// First will have to take in a number ` ` and return an array of length ` ` containing the numbers of the fibonacci sequences
+// I will also use the `for loop` method to generate the rest of the sequence by adding the last two numbers in the squesnce and pushing the sum onto the result array. 
```

Comment:
Great! I think a for loop is a good approach especially with the logic you are implementing in adding the last two numbers .